cluster/gce/config-common.sh

```diff
@@ -79,7 +79,7 @@ function get-master-disk-size() {
 
 function get-node-ip-range {
   if [[ -n "${NODE_IP_RANGE:-}" ]]; then
-    >&2 echo "Using user provided NODE_IP_RANGE: ${NODE_IP_RANGE}"
+    echo "Using user provided NODE_IP_RANGE: ${NODE_IP_RANGE}"
```

Comment:
The line below is probably not needed then...

Reply:
Whoops :smile: sorry about that. Fixed.

Reply:
Apparently, the stdout of this function is used [here](https://github.com/kubernetes/kubernetes/blob/0f2cccc98c3bf2a512ec55cbeba89dbfe7c768cf/cluster/gce/config-default.sh#L138), so we cannot remove `echo "${NODE_IP_RANGE}"` line. We cannot print `"Using user provided NODE_IP_RANGE: ${NODE_IP_RANGE}"` to stdout as well. After offline discussion with @mm4tt we decided to get rid of the stderr echo altogether.